oak_runtime/src/node/wasm/mod.rs

```diff
@@ -847,9 +847,8 @@ fn wasm_node_privilege(wasm_module_bytes: &[u8], signatures: &[Signature]) -> No
     let mut integrity_tags = hashset! { hash_tag };
 
     for signature_item in signatures.iter() {
-        let signature_tag = oak_abi::label::web_assembly_module_signature_tag(
-            &signature_item.public_key.to_bytes(),
-        );
+        let signature_tag =
+            oak_abi::label::web_assembly_module_signature_tag(&signature_item.public_key);
         confidentiality_tags.insert(signature_tag.clone());
```

Comment:
(pre-existing) Quick question to check my understanding: would we expect this secrecy label to normally use disjunctions rather than (as here) conjunctions, when disjunctions are available?  

AIUI this currently builds a label like `code:hash ∧ pubkey:A ∧ pubkey:B ∧ pubkey:C`, meaning that the client has to specify *all* of those tags.

Reply:
Note that this function computes the **privilege** of the node, i.e. the **maximum authority** that the node can use in declassifying (or endorsing) data.

This means that, for instance, a node with privilege `code:hash ∧ pubkey:A ∧ pubkey:B ∧ pubkey:C` may declassify **any of** those tags; so data labelled with a confidentiality label of `pubkey:A` (alone) may be declassified by this node, and so can data labelled with `pubkey:B`, etc.

Please do let me know if it is still unclear

Reply:
Doh, it's a privilege not a label &ndash; apologies for not reading closely enough.